internal/graph/graph.go

```diff
@@ -420,6 +421,40 @@ func newTree(prof *profile.Profile, o *Options) (g *Graph) {
 	return selectNodesForGraph(nodes, o.DropNegative)
 }
 
+// SimplifyFunctionNames replaces function names in graph with shortened names.
```

Comment:
No tests?..

Reply:
Added test -- unfortunately no tests for the flame graph, though.